spec/features/admin/accounting/transaction_flow_spec.rb

```diff
@@ -250,6 +250,45 @@
       click_on txn.txn_date.strftime("%B %-d, %Y")
       expect(page).to have_content("icecream")
     end
+
+    describe "an admin can edit a managed non-interest txn" do
+      let!(:txn) do
+        create(:accounting_transaction,
+               project_id: loan.id, description: "I love icecream", division: division, managed: true)
+      end
+
+      scenario do
+        visit admin_loan_tab_path(loan, tab: "transactions")
+        click_on txn.txn_date.strftime("%B %-d, %Y")
+        expect(page).to have_content("Edit")
+      end
+    end
+
+    describe "an admin cannot edit a non-managed txn" do
+      let!(:txn) do
+        create(:accounting_transaction,
+               project_id: loan.id, description: "I love icecream", division: division, managed: false, loan_transaction_type_value: :disbursement)
+      end
+
+      scenario do
+        visit admin_loan_tab_path(loan, tab: "transactions")
+        click_on txn.txn_date.strftime("%B %-d, %Y")
+        expect(page).not_to have_content("Edit")
+      end
+    end
+
+    describe "an admin cannot edit a managed interest txn" do
```

Comment:
I'm not sure if this spec is necessary since the above two already demonstrate that the policy is being applied correctly. The details of the policy fall to the policy spec. Or am I missing something?

Reply:
i'd rather have this redundancy here than not, since it's been broken, and the relationship between managed and transaction types is undertested throughout and underdocumented in the code, and we'll be changing things.